lighthouse-core/test/gather/driver-test.js

```diff
@@ -913,6 +913,64 @@ describe('.goOnline', () => {
   });
 });
 
+describe('.clearDataForOrigin', () => {
+  it('only clears data from certain locations', async () => {
+    let foundStorageTypes;
+    connectionStub.sendCommand = createMockSendCommandFn()
+      .mockResponse('Storage.clearDataForOrigin', ({storageTypes}) => {
+        foundStorageTypes = storageTypes;
+      });
+    await driver.clearDataForOrigin('https://example.com');
+    expect(foundStorageTypes).toMatchInlineSnapshot(
```

Comment:
can you add a comment for future editors of this file for the types they should be suspicious of if they appear in this snapshot? and/or could also outright assert the new important storage types aren't in here